.cursor/rules/code-review.mdc

```diff
@@ -0,0 +1,28 @@
+---
+description: Code Review Rules
+alwaysApply: false
+imports:
+  - git-commit-format
+  - 100-go-mistakes
+---
+
+Prior submit or push the code to Github, this steps should be reviewed
+
+Reviewing the introduced changes
+- Use the rules described in this place https://hypershift.pages.dev/contribute/#prior-to-submitting-a-pull-request in order to analise the code changes.
+
+Generate a mermaid diagram with the introduced code
+- if you add enumerations, remove the . and change them to -
+- Always in english
+- Avoid using {} or () in the names
+
+Generate a commit message according the code changed in the PR:
+- All commit messages should follow the format outlined in the git-commit-format rule.
+
+Basic Rules
+- Keep you honest, if a code change does not makes sense, say it
+- Don't give me the reason if I don't have it
+- Follow the 100-go-mistakes rules
+
+How code should be scoped
+- Follow the https://hypershift.pages.dev/contribute/ documentation
```

Comment:
```suggestion
- Follow the https://hypershift.pages.dev/contribute/documentation
```

Reply:
that url does not exists, changing the format.